src/System.Management.Automation/engine/remoting/client/ClientRemotePowerShell.cs

```diff
@@ -951,7 +951,7 @@ public void Dispose()
         /// Release all resources.
         /// </summary>
         /// <param name="disposing">If true, release all managed resources.</param>
-        protected void Dispose(bool disposing)
```

Comment:
This class is not meant to be derived from.  Since it is internal access, it can be made sealed without causing a breaking change.

Reply:
@PaulHigin Submitted a PR to seal the class: #15802.

Reply:
> it can be made sealed without causing a breaking change.

Completed in https://github.com/PowerShell/PowerShell/pull/15802.